chainer/datasets/fashion_mnist.py

```diff
@@ -8,6 +8,23 @@
 from chainer.datasets._mnist_helper import preprocess_mnist
 
 
+def fashion_mnist_label(label):
+    """Given a fashion-MNIST label, returns the name of that label
+
+    Args:
+        label (int): Label of image value, number from 0-9
+
+    Returns:
+       String value of English name of image label
+
+    """
+
+    assert range(10).index(label) >= 0
+
+    return ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal',
```

Comment:
Please define list of labels outside the function to avoid creating list every time.